villageline/settings/local.py.example

```diff
@@ -0,0 +1,8 @@
+from settings import *
+
+# SECRET_KEY =
```

Comment:
Perhaps this should be:

```suggestion
# DJANGO_SECRET_KEY =
```

Reply:
Agh, I was wrong about this - I was getting an error about the `DJANGO_SECRET_KEY` environment variable because it's still present in settings.py, but needs removing: 

https://github.com/richardnpaul/communityline/blob/bd30512d362d6f78b2526fb2f9f35a3f9370a475/villageline/settings/settings.py#L23

Reply:
You should be able so see that this is resolved now.  Not sure what happened, I also managed to fail to push a commit with fixes :weary: